composer.json

```diff
@@ -0,0 +1,62 @@
+{
+    "name": "synolia/sylius-admin-notification-plugin",
+    "type": "sylius-plugin",
+    "keywords": ["sylius", "sylius-plugin", "notification"],
+    "description": "Admin notifications for Sylius.",
+    "license": "EUPL-1.2",
+    "require": {
+        "php": "^7.4 || ^8.0",
+        "ext-json": "*",
+        "sylius/sylius": "^1.9",
+        "symfony/cache": "^5.4",
+        "symfony/notifier": "^5.4",
+        "symfony/monolog-bundle": "^3.0"
```

Comment:
Is it mandatory ? As your handler is not automatically registered in monolog conf, I would instead move this package in `suggest` part. 
For your handler, register the service in your DependencyInjection only if the monolog-bundle is present :)